ts/Extensions/Boost/BoostSeries.ts

```diff
@@ -954,11 +958,29 @@ function scatterProcessData(
             processedData.push({ x, y });
             processedXData.push(x);
             processedYData.push(y);
+            xDataMax = Math.max(xDataMax, x);
+            xDataMin = Math.min(xDataMin, x);
+            yDataMax = Math.max(yDataMax, y);
+            yDataMin = Math.min(yDataMin, y);
         } else {
             cropped = true;
         }
     }
 
+    if (!isNumber(xExtremes.max) || !isNumber(xExtremes.min)) {
+        xAxis.setOptions({
+            max: xAxis.options.max || xDataMax,
```

Comment:
If the check were to be done before the calculation of the xDataMax then it should make the boost run even faster when the axis extremes are set. Should we have some performance tests similar to e.g. the ones for the Stock?

Reply:
(preferably in a different task but that would help with comments like mine here)

Reply:
Performance tests are a good idea, especially for boost.